usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -104,6 +106,16 @@
         "custom_schema": "recipient_hash STRING",
         "column_names": list(RECIPIENT_PROFILE_COLUMNS),
     },
+    "summary_state_view": {
```

Comment:
I think we would want this to live in the `load_query_to_delta` in since it isn't a direct copy of a Postgres table.